docs/src/plotting.md

```diff
@@ -0,0 +1,153 @@
+# Confusion matrices
+
+```@docs
+Lighthouse.plot_confusion_matrix
+```
+
+```@setup 1
+using CairoMakie
+CairoMakie.activate!(type = "png")
+
+```
+
+```@example 1
+using Lighthouse: plot_confusion_matrix, plot_confusion_matrix!
+confusion = [0.1 0.2 0.3 0.4 0.0;
+        0.1 0.2 0.3 0.4 0.5;
+        0.1 0.2 1.0 0.4 0.5;
+        0.1 0.2 0.3 0.4 0.5;
+        0.9 0.2 0.3 0.4 0.5]
+classes = ["class $i" for i in 1:size(confusion, 1)]
+fig, ax, p = plot_confusion_matrix(confusion, classes, :Row)
+```
+
+```@example 1
+fig = Figure(resolution=(800, 400))
+plot_confusion_matrix!(fig[1, 1], confusion, classes, :Row, annotation_text_size=14)
+plot_confusion_matrix!(fig[1, 2], confusion, classes, :Column, annotation_text_size=14)
+fig
+```
+
+# Reliability calibration curves
+
+```@docs
+Lighthouse.plot_reliability_calibration_curves
+```
+
+```@example 1
+using Lighthouse: plot_reliability_calibration_curves
+
+curves = [(LinRange(0, 1, 10), range(0, stop=i/2, length=10) .+ (randn(10) .* 0.1)) for i in -1:3]
+
+plot_reliability_calibration_curves(
+    curves,
+    rand(5),
+    classes
+)
+```
+
+# PRG curves
+
+```@docs
+Lighthouse.plot_prg_curves
+```
+
+```@example 1
+using Lighthouse: plot_prg_curves
+plot_prg_curves(
+    curves,
+    rand(5),
```

Comment:
this should maybe be a StableRNG (or the global rng should be seeded at the start of the docs) to make sure docs always generate the same plots?